external/CMakeLists.txt

```diff
@@ -31,9 +31,9 @@ if (DEFINED SPIRV-Headers_SOURCE_DIR)
   set(SPIRV_HEADER_DIR ${SPIRV-Headers_SOURCE_DIR})
 else()
   if (IS_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/SPIRV-Headers)
```

Comment:
condition is wrong no?

Reply:
Yes, it is.

Reply:
Unless I'm reading it wrong:
```
 if (IS_DIRECTORY uppercase_version) {
    set_path_to_lowercase
 } else {
    set_path_to_uppercase
 }
 ```
 
 cloning the PR locally fails to build (as the CI does), because the uppercase folder is not found, but selected.